Content.Client/Decals/DecalSystem.cs

```diff
@@ -61,6 +61,7 @@ public override void Shutdown()
         protected override bool RemoveDecalHook(GridId gridId, uint uid)
         {
             RemoveDecalFromRenderIndex(gridId, uid);
+            ChunkIndex[gridId].Remove(uid);
```

Comment:
this is already done through removedecalinternal which should be called as opposed to this but ig it should be fine?